thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -696,15 +696,25 @@ export default Controller.extend({
      * @param {Object} rangeOption - the selected range object
      */
     onRangeOptionClick(rangeOption) {
```

Comment:
Making this action more consistent with the manage.alerts.tune action for the same component (date-range-picker)